# `integers.txt`から最初の数値を読み取り、
# それに対して何らかの計算を実行します。
# 考えられる少なくとも2つの例外(`IOError`と`ValueError`)を
# `except`文でキャッチできるようにし、
# いずれのエラーも該当しない場合にのみ計算を続行します。

file_name = 'integers.txt'
